facebook/delphiFacebook/R/responses.R

```diff
@@ -667,9 +701,11 @@ filter_complete_responses <- function(data_full, params)
   data_full <- select(data_full, -.data$zip5)
 
   # 9 includes StartDatetime, EndDatetime, Date, token, wave, geo_id,
-  # UserLanguage + two questions (ignore raceethnicity field which may or may
-  # not exist, depending on params)
-  valid_row_filter <- rowSums( !is.na(data_full[, names(data_full) != "raceethnicity"]) ) >= 9
+  # UserLanguage + two questions (ignore raceethnicity, module, and
+  # w12_assignment fields which may or may not exist, depending on params and
+  # survey version)
+  ignore_cols <- c("raceethnicity", "w12_assignment", "module")
+  valid_row_filter <- rowSums( !is.na(data_full[, !(names(data_full) %in% ignore_cols)]) ) >= 9
```

Comment:
Hm, you're adding both `w12_assignment` and `module` here, but I thought `module` was also in Wave 11. Were we applying the inclusion criteria incorrectly for Wave 11 before this?

Reply:
Yeah, with the previous version of the filter Wave 11 responses would have only required 1 answered question, so we'll need to reissue those microdata (or run the existing files through just this corrected filter?).